.github/workflows/build_deploy_docs_develop.yml

```diff
@@ -49,7 +49,7 @@ jobs:
 
       - name: Build html
         run:
-          poetry run portray as_html --overwrite -o site
+          poetry run mkdocs build -f ./docs/mkdocs.yml -d ../site"
```

Comment:
Is the trailing `"` correct? seems rather odd

Reply:
You're right ofcourse